Safety/emergency-response-guide.rst

```diff
@@ -0,0 +1,173 @@
+This does not replace safety training required to be on the summit.
+It is recommended to contact the safety team to complete summit driver's training in addition to the required summit training. 
+
+.. note::
+    Observing specialists serve as the Nighttime Safety Officers if there is not a member of the safety team physically on the summit.
+    If they take the decision to close the telescope or evacuate the summit for inclement weather, that decision is not negotiable.
+    The priority of safety will always be given to staff first, equipment second. 
+
+.. _emergency-response-guide-medical-emergency:
+
+Medical Emergency
+^^^^^^^^^^^^^^^^^
+
+.. Preconditions
+
+Contact the PolyClinic Paramedic at extension 577
+
+.. note::
+    If no one answers the telephone, use any Rubin radio tuned to Channel 1. Put out the call:
+    
+    - "Atento paramedico, habla [your name]."
+    Wait for a response.
+    If there is no response on the radio, call 440 to the Mountain Assistant.
+
+Describe the emergency:
+
+- Type of Emergency
+
+    My collegue is unconscious.
+        - "Mi colega esta inconsciente."
+
+    My collegue is not breathing.
+        - "Mi colega no respira."
+
+    My colleague has fallen.
+        - "Mi colega se ha caído."
+
+    My colleague has fainted.
+        - "Mi colega se ha desmayado."
+
+    My colleague is bleeding profusely.
+        - "Mi colega esta sangrando mucho."
+
+- Your name
+
+    My name is [your name].
+        - "Me llamo [your name]."
+
+- Your location
+
+    We are in [AuxTel/Main Building] [floor 1, 2, 3, ..].
+        - "Estamos en el [telescopio auxilliar/edificio principal] de Rubin, en el piso [uno, dos, tres, ...]."
+
+- How many people are injured
+
+- Name any hazardous conditions present IF you can identify them. 
+  Speak only to the details you know.
+
+- Wait for instructions
+
+.. _emergency-response-guide-fire:
+
+Fire
+^^^^
+
+- If you know of anyone else in the building not with you, use a Rubin radio tuned to Channel 3. 
+  
+  Attention Rubin Personnel, FIRE!
+  Evacuate the building!
+  
+  "Atento a todo personal, HAY INCENDIO!
+  Salgan del edificio!"
+
+- Give everyone you encounter directions to meet at the emergency assembly point in the first level parking lot.
+
+- Pull a fire alarm if possible.
+
+- Call the Mountain Assistant at extension 440:
+    - If they do not answer, call Control Puerta at extension 434.
+    - Put out a call for help on any Rubin radio tuned to Channel 1. 
+
+- Use a fire extinguisher if you are trained to do so.
+
+- Remain low to the ground if you encounter smoke.
+
+- Leave all belongings behind.
+
+- Take the stairs, not the elevators.
+
+.. _emergency-response-guide-earthquake:
+
+
+Earthquake
+^^^^^^^^^^^
+
+- Drop, cover, and remain still.
+
+- Stay away from windows and any electrical panels.
+
+- After the shakes stop, give directions to meet at the emergency assembly point at first level parking.
```

Comment:
shaking

Reply:
Is this a function of severity? 
Clearly for small ones no one does anything. 
In some ways, if people feel the need to go under a desk then it's probably worth meeting at an assembly point etc. I'm not if there is an official guideline there?

Reply:
This text is taken directly from Giovanni's safety guide. The safety point is the first level parking area outside of the building. I have clarified these steps in the severity of the earthquake.